src/wallet/rpcdump.cpp

```diff
@@ -84,7 +84,7 @@ UniValue importprivkey(const UniValue& params, bool fHelp)
     if (!EnsureWalletIsAvailable(fHelp))
```

Comment:
This is one of the other uses of `fHelp`. In this case, I it determines the behavior when the wallet is disabled. If `fHelp`, it returns `false`, otherwise it throws. I don’t really understand why, though, because in the `fHelp` case, we just return JSON `null`, not actually give them a help message (with potentially a note that the node queried doesn’t support that operation).

So the change is that now sending the wrong number of arguments would result in a `null` response rather than getting to the help message. I _think_ the solution is to fix `EnsureWalletIsAvailable` to not rely on `fHelp` and never throw, but to simply result in a help message (again, with a note). So changing the lines below to like
```c++
if (EnsureWalletIsAvaliable() || fHelp)
    throw runtime_error(
        (EnsureWalletIsAvaliable() ? "" : "NOTE: This node does not have wallet support.\n" +
```

Reply:
This should be addressed in #6473.